app.js

```diff
@@ -1,9 +1,26 @@
 import express from "express";
+import cors from "cors";
+
+import { AuthorizationController } from "./controllers/authorization.controller.js";
+
+import { checkToken } from "./middleware/token.middleware.js";
+import { WeatherController } from "./controllers/weather.controller.js";
+
+const { CORS_ORIGIN } = process.env;
 
 const app = express();
 
-app.get("/", (req, res) => {
-  res.send("works");
-});
+app.use(
+  cors({
+    credentials: true,
+    origin: CORS_ORIGIN,
+  }),
+);
+
+app.get("/authorization", checkToken, AuthorizationController.authCheck);
+
+app.post("/login", AuthorizationController.login);
+
+app.get("/weather", checkToken, WeatherController.getWeather);
```

Comment:
This endpoint receives a city as a query param. Without the city this endpoint couldn't work, so it is better to rename it to the getWeatherForCity, and add parameter validation